analytical/templatetags/google_analytics_gtag.py

```diff
@@ -51,8 +55,18 @@ def __init__(self):
             "must be a string looking like 'UA-XXXXXX-Y'")
 
     def render(self, context):
+        other_fields = {}
+
+        identity = get_identity(context)
+        if identity is not None:
+            other_fields['user_id'] = identity
+
+        extra = '\n'.join([
+            GTAG_SET_CODE.format(key=key, value=value) for key, value in other_fields.items()
+        ])
         html = SETUP_CODE.format(
             property_id=self.property_id,
+            extra=extra
```

Comment:
A trailing comma might be nice on this line.

Reply:
Done, thanks.